lib/common.sh

```diff
@@ -17,7 +17,20 @@ detect_sbt() {
 }
 
 is_play() {
-  _has_playConfig $1
+  local app_dir=$1
+
+  case "${IS_PLAY_APP}" in
+  "true")
+    return 0
+    ;;
+  "false")
+    return 1
+    ;;
+  *)
+    [[ -f "${app_dir}/${PLAY_CONF_FILE:-conf/application.conf}" ]] ||
+      grep -E --quiet --no-messages '^\s*addSbtPlugin\(("org\.playframework"|"com\.typesafe\.play")\s*%\s*"sbt-plugin"\s*%\s*".*?"\)\s*?$' "${app_dir}/project/plugins.sbt"
```

Comment:
I think this regex misses the following cases (which might be expected, but mentioning in case you wanted to support them, particularly given this new implementation is now more strict so could break some customers if they are doing these things):

```
// whitespace before the first quote
addSbtPlugin( "com.typesafe.play" % "sbt-plugin" % "2.4.2")

// whitespace before the after the final quote
addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.4.2" )

// comment at end of line
addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.4.2") // some comment
```

It seems the later parts of the regex (the ones that match against the version number and closing parens etc) are not really required, and could be dropped to (a) simplify the regex, (b) make it easier to support the "comment at end of line" case.

I think this would support all of the above:

```suggestion
      grep -E --quiet --no-messages '^\s*addSbtPlugin\(\s*("org\.playframework"|"com\.typesafe\.play")\s*%\s*"sbt-plugin"' "${app_dir}/project/plugins.sbt"
```

Lastly, I presume single quotes aren't supported in the`addSbtPlugin` usages in `plugins.sbt`? It wasn't clear to me what syntax this file uses - is it Scala or a custom syntax? If Scala then searching now I see single quotes are chars instead, so the regex wouldn't need to support them.

Reply:
Yeah, no single quotes allowed. :) Good point about the ending - I will adjust the regex accordingly.